src/marketplace/sellers/marketing-review-guidelines.md

```diff
@@ -9,32 +9,83 @@ From a marketing perspective, all submissions to Magento Marketplace must solve
 
 We have made an investment in the presentation of Magento Marketplace. In return, we ask that you put forth an effort to present your solutions in a professional manner. If we see room for improvement, we’ll reach out to you before listing your extension on Magento Marketplace.
 
--  Your product must have a professional presentation with clear descriptions, images, and screenshots.
--  The product name should indicate the purpose of the extension or theme.
--  The product description should be well-written, and provide an overview of the features and benefits.
--  Use only high-quality screenshots, images, and video to accompany the listing.
+*  Your product must have a professional presentation with clear descriptions, images, and screenshots.
+*  The product name should indicate the purpose of the extension or theme.
+*  The product description should be well-written, and provide an overview of the features and benefits.
+*  Use only high-quality screenshots, images, and video to accompany the listing.
+
+## Extension title
+
+The extension title should fulfill the following guidelines:
+
+*  Choose a title that is descriptive and concise. The extension title can be up to five words long and must be in English.
+*  Do not include the version number, developer name, or any reference to Magento in the title.
+   *  Do not use _"Magento"_ or _"M2"_.
+*  Do not include the words _"Extension"_, _"Module"_ or _"Plugin"_.
+   *  However, the words, _"Connector"_ and _"Integration"_ can be used if applicable.
+*  Use _"&"_, instead of _"And"_. (_"Order & Save"_, not _"Order and Save"_)
+*  First letter of each word capitalized. (_"Mass Order Export"_, not _"Mass order export"_)
+   *  Do not use All Caps.
+*  No punctuation unless it’s part of the actual company name. (_"InstantSearch+"_, _"Yahoo!"_)
+   *  _"/"_ and _"-"_ are acceptable in certain circumstances. (_"Two-Factor"_, _"Import / Export"_)
+*  Not formatted like package name with underscores between words. (_"Extension_Title"_)
+*  No Developer Name in the title. (_"Print Order PDF"_, not _"Fooman Print Order PDF"_)
+   *  EXCEPTION: Integration. If the developer is the same name as the service being integrated. For instance, if Visa submits an integration, they SHOULD include _"Visa"_ since it’s the name of the service.
```

Comment:
```suggestion
   *  If the developer is the same name as the service being integrated, you may use the develop name in the title. For instance, if Visa submits an integration, they should include _Visa_ since it’s the name of the service.
```